### ISSUE TITLE
Source code?

### ISSUE THREAD

**Issue by rollmeister** [NerdMosby/minerd-LE#2]
I wanted to add sha256 acceleration for armv8 devices. Got code ready for it.

**Comment by babyrig** [comment]
I hope you can do it I have an armv8 devices too :)

**Comment by rollmeister** [comment]
> I hope you can do it I have an armv8 devices too :)
I might be able to add it to the wallet/miner repo at https://github.com/Bitcoin-LE/bitcoinle-core. I am asking the developer to hint me which file needs the changes done on. I suggest you monitor the issues tab there for updates where I will announce it, if I get it working.

**Comment by rollmeister** [comment]
I have made initial attempts to this. https://github.com/rollmeister/bitcoinle-core-armv8 scroll down and refer to README.md there for important notes.

**Comment by babyrig** [comment]
Hello dev, this binaries have some included fee or anything because I can't find any information about it !

thanks

**Comment by rollmeister** [comment]
> Hello dev, this binaries have some included fee or anything because I can't find any information about it !
> 
> thanks

Yes there is a dev fee. A percentage of the pool miner work is donated to the developer. How much, nobody knows. There will be no source code for a while. I am still holding out on armv8 binaries though. Not hard to do actually.
